legacy/documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -179,11 +179,11 @@ Configuration is minimal.
 Add the configuration properties to your `persistence.xml` or where you put the rest of your Hibernate Search configuration.
 
 Select Elasticsearch as the backend:: `hibernate.search.default.indexmanager elasticsearch`
-Hostname and port for Elasticsearch:: `hibernate.search.default.elasticsearch.host \http://127.0.0.1:9200` (default)
```

Comment:
Please don't change the legacy documentation... ?

Reply:
Argh, silly me, will fix.

Reply:
Fixed.